pkg/proxy/proxy.go

```diff
@@ -410,6 +410,9 @@ func (p *Proxy) ReinstallRoutingRules() error {
 			return err
 		}
 
+		if err := removeFromEgressProxyRoutesIPv4(); err != nil {
```

Comment:
Does this mean that this change, and the subsequent one to add the egress proxy rules must be in introduced in different patch versions?

Reply:
Yes, will be backported to all stable branches.